src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,362 @@
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public PossibleAction getAction(PossibleAction lastAction, Player activePlayer, Player nonActivePlayer) {
+        System.out.println(activePlayer.getStackPlayer() + " " + activePlayer.getNamePlayer());
+        System.out.println(nonActivePlayer.getStackPlayer() + " " + nonActivePlayer.getNamePlayer());
+        System.out.println(activePlayer.getNamePlayer() + " decyduje");
+        System.out.println("Podaj decyzje");
+        Scanner scanner = new Scanner(System.in);
+        int decision;
+        if (lastAction == null && gameState.getGamePhase() == GamePhase.PREFLOP) {
+            System.out.println("1 limp 2 raise");
+            decision = scanner.nextInt();
+            if (decision == 1) {
+                return PossibleAction.Limp;
+            } else {
+                return PossibleAction.Raise;
+            }
+        } else if (lastAction == null && gameState.getGamePhase() != GamePhase.PREFLOP) {
+            System.out.println("1 check 2 bet");
+            decision = scanner.nextInt();
+            if (decision == 1) {
+                return PossibleAction.Check;
+            } else {
+                return PossibleAction.Bet;
+            }
+        } else if (lastAction == PossibleAction.Check) {
+            System.out.println("1 check 2 bet");
+            decision = scanner.nextInt();
+            if (decision == 1) {
+                return PossibleAction.Check;
+            } else {
+                return PossibleAction.Bet;
+            }
+        } else if (lastAction == PossibleAction.Bet || lastAction == PossibleAction.Raise) {
+            System.out.println(amountToCall(nonActivePlayer, activePlayer) + "$ to call");
+            System.out.println("1 Call, 2 Raise, 3 Fold");
+            decision = scanner.nextInt();
+            if (decision == 1) {
+                System.out.println(activePlayer.getNamePlayer() + " calls $" + amountToCall(nonActivePlayer, activePlayer));
+                return PossibleAction.Call;
+            } else if (decision == 2) {
+                return PossibleAction.Raise;
+            } else {
+                return PossibleAction.Fold;
+            }
+        } else {
+            System.out.println("1 check, 2 raise");
+            decision = scanner.nextInt();
+            if (decision == 1) {
+                return PossibleAction.Check;
+            } else {
+                return PossibleAction.Raise;
+            }
+        }
+    }
+
+    public void handleAction(PossibleAction lastAction, Player activePlayer, Player nonActivePlayer) {
+        if (lastAction == PossibleAction.Limp) {
+            System.out.println(activePlayer.getNamePlayer() + " limps 0,5$");
+            updatePotAndStackAmount(gameState.smallBlind, activePlayer);
+            activePlayer.increasePlayerMoneyAtStrit(gameState.smallBlind);
+            firstDecisionSmallBlind = false;
```

Comment:
wyciagnij to do funkcji handleLimp() (bo np handleRaisePlayer() i handleBetPlayer() wyciagnales)